docs/examples/webextension/background.js

```diff
@@ -0,0 +1,12 @@
+var TESTPILOT_TELEMETRY_CHANNEL = 'testpilot-telemetry';
+var testpilotPingChannel = new BroadcastChannel(TESTPILOT_TELEMETRY_CHANNEL);
+setInterval(function () {
+  testpilotPingChannel.postMessage({
+    boolData: true,
+    arrayOfData: ["one", "two", "three"],
+    nestedData: {
+      intData: 10,
+    },
+  });
```

Comment:
Same as the other mechanism - just a JS object with what you want included in the payload of the Telemetry ping. Test Pilot itself is agnostic on this, so it's more a concern between you & the metrics team to spec out.